### PR TITLE
Change to current hydrator version

### PR DESCRIPTION
**Pull request by tivv**

### CODE REVIEW

**Review by dli357 — APPROVED** [review]
LGTM

**Review by dli357 — COMMENTED** [review]
Seems like Github Actions has a check failure, please resolve it before merging if it's related to your change, thanks!

**Comment by tivv** [comment]
That's a known problem,  I am looking into it separately. I will merge this to unblock BUT

**Pull request merged**
